deploy/terraform-datacommons-website/examples/setup/variables.tfvars

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ # The follow fields must be filled out before running the setup script.
+
+# GCP project id where you wish to set up the Data Commons instance in.
+# Ideally this is a new GCP project dedicated to Data Commons.
+#
+# Please consult the doc below on how to create one,
+# or reach out to your GCP org admin to have one created.
+# https://developers.google.com/workspace/guides/create-project
+project_id="Replace me"
+
+
+# The setup script will create a new domain that will host the Data Commons
+# web application. As a part of the standard domain registration process,
+# the script requires the domain you wish to create, and the contact information
+# associated with the domain. The information is used for registering the domain,
+# and will not be publicly available.
+
+# The domain you wish to create. During the installation step, the Data Commons
+# web application will be installed under this domain.
+#
+# Please not that this domain must be available, or else the script will fail.
+# To check if the domain is available, please search it up using the link below.
+# https://domains.google/
+# If you see a green checkmark in the search results, it means that is available.
+# Please also fill out the associated price for the domain you see on the search
+# result. It must be an exact match in the format of "<amount 2 decimal> <currency".
+# Ex: "12.00 USD"
+dc_website_domain="Replace me"
+domain_yearly_price="Replace me"
+
+# E-mail where you can be reached. For the domain to be active,
+# you must click on the actination link sent to this email after
+# running this script.
+contact_email="Replace me"
+
+# Contact phone number in international format.
+# Please consult the link below as needed.
+# https://en.wikipedia.org/wiki/National_conventions_for_writing_telephone_numbers
+# Example for USA +1.6502530000
+contact_phone_internaltional_format="Replace me"
+
+# Region is the country equivalent of the contact address.
+# Ex: US for United States, CA for Canada
+contact_region_code="Replace me"
```

Comment:
Shall we put all the "Replace me" here with a meaning default value, so if use changes nothing, the whole script can still run?